requirements.txt

```diff
@@ -1,10 +1,4 @@
-# This pulls in all the libraries needed to run Selenium tests
-# on Mozilla WebQA projects
-
-py==1.4.14
-pytest==2.3.5
-pytest-xdist==1.8
-unittestzero
-requests==2.4.3
+pytest==2.8.0
+pytest-xdist==1.13.1
```

Comment:
I missed this earlier (or perhaps ignored) but we've been removing pytest-xdist from our project requirements as it's not strictly a requirement but instead a productivity boost. If we remove this however, we'll be pinned to an earlier version of pytest-xdist on our CI that appears to not be compatible with pytest 2.8 so my recommendation would be to remove this line and pin pytest to 2.7.2. This will also address issue #70.